src/ggml.c

```diff
@@ -12012,6 +11909,116 @@ static void ggml_compute_forward_mul_mat(
     }
 }
 
+// GEMM
+// TODO: compare gemm op with the current implementation of mul_mat
+
+static void ggml_compute_forward_mul_mat_f16_f32(
+        const struct ggml_compute_params * params,
+        const struct ggml_tensor * src0,
+        const struct ggml_tensor * src1,
+              struct ggml_tensor * dst) {
```

Comment:
This seems unnecessary - try to remove this and just reuse the original function